hello-world/README.md

```diff
@@ -0,0 +1,64 @@
+## Conclave Sample
```

Comment:
Can we give a better name to this sample besides "hello world"? How about enclave-persistence (for the project name)?